apps/api/src/app/import/import.controller.ts

```diff
@@ -35,17 +36,14 @@ export class ImportController {
 
   @Post()
   @UseGuards(AuthGuard('jwt'))
+  @HasPermission(permissions.createAccount)
   @UseInterceptors(TransformDataSourceInRequestInterceptor)
   @UseInterceptors(TransformDataSourceInResponseInterceptor)
   public async import(
     @Body() importData: ImportDataDto,
     @Query('dryRun') isDryRun?: boolean
   ): Promise<ImportResponse> {
     if (
-      !hasPermission(
-        this.request.user.permissions,
-        permissions.createAccount
-      ) ||
       !hasPermission(this.request.user.permissions, permissions.createOrder)
```

Comment:
I would change this to `permissions.createAccount`

Reply:
OK, got it :-)